src/lister_haxe/lister.hx

```diff
@@ -0,0 +1,38 @@
+import haxe.io.Path;
+using StringTools;
+
+class Lister {
+  static public function get(url:String):Array<String> {
+    var http = new haxe.Http(url);
+    var hits = [];
+
+    http.onData = function(resp) {
+      //trace(resp);
+      var rgx = ~/<img[^>]+src=([^ >]+)/s;
+
+      while (rgx.match(resp)) {
+        hits.push(clean(rgx.matched(1), url));
+        resp = rgx.matchedRight();
+      }
+    };
+
+    http.onError = function(err) {
+      trace('ERROR: $err');
+
+      throw err;
```

Comment:
Like in javascript, Haxe lets you throw anything.  I'm curious to see what this does when compiled to C#, especially if you throw something exotic like a function or a regex.  :)